app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/snippet/SnippetsFragment.kt

```diff
@@ -23,16 +19,6 @@ class SnippetsFragment : BaseDataBindingFragment<SnippetsFragmentViewModel, Snip
         binding.apply {
             snippetList.adapter = GroupAdapter<GroupieViewHolder>().apply {
                 add(fragmentViewModel.snippetGroup)
-                setOnItemClickListener { item, _ ->
```

Comment:
@ianslinker-br Was this code doing anything before it was removed? If so, please add a TODO to add click handling back in the future.

Reply:
This code was checking if the view models were of type RepoFileViewModel when the view model only ever generated SnippetViewModels for the adapter. My guess is, this screen was copied from RepositoryFragment which had an identical block of code but was never updated for click code specific to snippets. As such I am not sure where, if anywhere, a snippet should navigate to when clicked and thus I removed the code.

Reply:
That definitely makes sense. Thanks for the explanation!